core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/mercury/streams/streams.go

```diff
@@ -208,6 +212,7 @@ func (s *streams) CheckCallback(ctx context.Context, values [][]byte, lookup *me
 func (s *streams) makeCallbackEthCall(ctx context.Context, payload []byte, lookup *mercury.StreamsLookup, checkResults []ocr2keepers.CheckResult, i int) error {
 	var responseBytes hexutil.Bytes
 	args := map[string]interface{}{
+		"from": zeroAddress,
```

Comment:
can you also update in line 310 here (in eth call we make to AllowedToUseMercury)

Reply:
we only check for this address in 3 functions: checkUpkeep, simulate and executeCallback.
should we populate this for other eth_call too?